Specify error for a DATA frame before any HEADERS frames (#2715)
quicwg/base-drafts: draft-ietf-quic-http.md

The "chunked" transfer encoding defined in Section 4.1 of {{!RFC7230}} MUST NOT
be used.

> If a DATA frame is received before a HEADERS frame on a request stream, the

Reviewer: ```suggestion
If a DATA frame is received before a HEADERS frame on a either a request or push stream, the
```

> recipient must respond with a connection error ({{errors}}) of type

Reviewer 2: ```suggestion
recipient MUST respond with a connection error of type HTTP_UNEXPECTED_FRAME
```

> HTTP_UNEXPECTED_FRAME.

Reviewer 2: ```suggestion
 ({{errors}}).
```
